docs/tutorial_sample_boundaries.md

```diff
@@ -0,0 +1,646 @@
+## Predicting sample boundaries
+
+![tutorial-goal](./figures/tomo_side_light.png)
+*Side view onto a cryo-electron tomogram [run 15094](https://cryoetdataportal.czscience.com/runs/15094) without (left) and with (right) sample boundary annotation*
+
+Biological samples acquired in a cryoET experiment are usually thin slabs of vitrified ice containing the biological specimen of interest. Unfortunately, it is difficult to determine orientation and thickness of the samples ahead of reconstruction. For this reason, volumes reconstructed from cryoET tilt series are often larger than the actual sample and contain a significant amount of empty space (i.e. the vacuum inside the TEM column).
+
+There are several reasons for why it can be useful to determine more accurate sample boundaries, for example:
+
+- statistical analysis of the sample preparation process
+- masking out the vacuum region to reduce the size of the volume
+- masking out the vacuum region during the training of a neural network
+- capping of membrane segmentations to define topological boundaries
+
+Below, we will show how to use [**copick**](https://github.com/copick/copick), an adapted version of [deepfinder](https://github.com/jtschwar/cryoet-deepfinder/tree/master) and [album](https://album.solutions/) to predict sample boundaries for datasets [10301](https://cryoetdataportal.czscience.com/datasets/10301) and [10302](https://cryoetdataportal.czscience.com/datasets/10302) from the [CZ cryoET Data Portal](https://cryoetdataportal.czscience.com). Copick is a cross-platform, storage-agnostic and server-less dataset API for cryoET datasets.
+
+![topview](./figures/tomo_top_both.png)
```

Comment:
I'm not sure if this is possible with our documentation tool, but would it be possible to center-justify all the figures? 

This shouldn't be a blocker, but it would make the page look prettier.

Reply:
That is possible, but not with using plain markdown, which is what I am familiar with. I have added it to the list of items for improving the doc site design. Melissa will be able to help with this.